src/sage/graphs/generic_graph.py

```diff
@@ -21014,6 +21016,95 @@ def layout_graphviz(self, dim=2, prog='dot', **options):
         positions = dot2tex.dot2tex(self.graphviz_string(**options), format='positions', prog=prog)
 
         return {key_to_vertex[key]: pos for key, pos in positions.items()}
+    
+    def layout_tutte(self, external_face, external_face_pos=None, **options):
+        """
+
+        Compute graph layout based on a Tutte embedding.
+
+        The graph must be 3-connected and planar.
+
+        INPUT:
+
+        - ``external_face`` -- list; the external face to be made a polygon
+
+        - ``external_face_pos`` -- list (default: ``None``); the positions of the vertices 
+          of the external face. If ``None``, will automatically generate a unit sided regular polygon.
+        
+        - ``**options`` -- other parameters not used here
+
+        OUTPUT: a dictionary mapping vertices to positions
+
+        EXAMPLES::
+
+            sage: g = graphs.WheelGraph(n=7)
+            sage: g.layout_tutte(external_face=[1, 2, 3, 4, 5, 6])
```

Comment:
This doctest is unstable. I obtain different results on my laptop (macOS...)
```
File "src/sage/graphs/generic_graph.py", line 21041, in sage.graphs.generic_graph.GenericGraph.layout_tutte
Failed example:
    g.layout_tutte(external_face=[1, 2, 3, 4, 5, 6])
Expected:
    {1: (-0.499999999999999, 0.866025403784438),
    2: (0.500000000000000, 0.866025403784439),
    3: (1.00000000000000, 8.85026869717109e-17),
    4: (0.500000000000000, -0.866025403784439),
    5: (-0.500000000000001, -0.866025403784438),
    6: (-1.00000000000000, 4.55896726715916e-16),
    0: (-5.55111512312578e-17, 1.86944233679563e-16)}
Got:
    {0: (4.16333634234434e-16, 7.42055745992141e-16),
     1: (-0.499999999999999, 0.866025403784439),
     2: (0.500000000000000, 0.866025403784440),
     3: (1.00000000000000, 7.31549942167514e-16),
     4: (0.500000000000000, -0.866025403784438),
     5: (-0.500000000000000, -0.866025403784437),
     6: (-1.00000000000000, 1.29124025055007e-15)}
```
You should find another test that is more stable. Printing vertex positions is never a good idea.

Reply:
In many of the other layout examples, they include this sort of example. However, I've removed it from this function.